lib/ansible/modules/storage/purestorage/purefa_volume.py

```diff
@@ -69,20 +73,37 @@
     eradicate: true
     state: absent
     fa_url: 10.10.10.2
+    api_token: e31060a7-21fc-e277-6240-25983c6c4592
+
+- name: Create clone of volume bar named foo
+  purefa_volume:
+    name: foo
+    target: bar
+    state: present
+    fa_url: 10.10.10.2
+    api_token: e31060a7-21fc-e277-6240-25983c6c4592
+
+- name: Overwrite volume bar with volume foo
+  purefa_volume:
+    name: foo
+    target: bar
+    overwrite: true
+    state: present
+    fa_url: 10.10.10.2
     api_token: e31060a7-21fc-e277-6240-25983c6c4592'''
 
 RETURN = '''
 '''
 
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.pure import get_system, purefa_argument_spec
+
```

Comment:
Note: this is just a style comment but the previous placement of this was correct.  This module is part of ansible, therefore, the purestorage import is an import from a third party lib and the ansible imports are from a local lib.  (See https://www.python.org/dev/peps/pep-0008/#imports )